flash_attn/models/llama.py

```diff
@@ -308,7 +309,28 @@ def config_from_meta_checkpoint(
         num_attention_heads=params["n_heads"],
         num_hidden_layers=params["n_layers"],
         rms_norm_eps=params["norm_eps"],
+        num_key_value_heads=params.get("n_kv_heads", None),
     )
+    multiple_of = params.get("multiple_of", 1)
+    ffn_dim_multiplier = params.get("ffn_dim_multiplier", None)
+
+    intermediate_size = 4 * config.hidden_size
```

Comment:
this could be due to me missing context here, but could we get some clarity on this `intermediate_size` computation logic? 

fwiw, it seems this would not give the result i expect for the old 7b model (intermediate_size = )

Reply:
This is from the llama code release:
https://github.com/facebookresearch/llama/blob/1a240688810f8036049e8da36b073f63d2ac552c/llama/model.py#L224
https://github.com/facebookresearch/llama/blob/1a240688810f8036049e8da36b073f63d2ac552c/llama/model.py#L195-L199

I've added comments to the code